### PR TITLE
Story/ece651 43 footer system testing

### PR DESCRIPTION
**Pull request by Riya0001**
Test cases created for footer

Test results location - Documents/General/Sprint 5/System Testing/ECE651-43

### CODE REVIEW

File: src/__tests__/footer.test.js
Comment (kar12thik) [inline src/__tests__/footer.test.js:11]: We do we need jest mocks for Footer?

File: src/__tests__/footer.test.js
Comment (kar12thik) [inline src/__tests__/footer.test.js:23]: Redundant function call needs to cleaned up.

**Review by kar12thik — COMMENTED** [review]
@Riya0001 Do not close the PR, address the changes requested in open PR.

**Comment by Riya0001** [comment]
Made the requested changes waiting for PR approval

File: src/__tests__/footer.test.js
Comment (Riya0001) [reply src/__tests__/footer.test.js]: Fixed

File: src/__tests__/footer.test.js
Comment (Riya0001) [reply src/__tests__/footer.test.js]: Fixed

File: src/__tests__/footer.test.js
Comment (kar12thik) [inline src/__tests__/footer.test.js:10]: This block needs to be removed as well, since no jest mocks being used.

Comment (Riya0001) [reply src/__tests__/footer.test.js]: Fixed

**Review by kar12thik — APPROVED** [review]

**Review by db8patel — APPROVED** [review]

**Review by MareenaFr — APPROVED** [review]
Looks good.

**Pull request merged**
